### ISSUE TITLE
Is it possible to run the frigate using my own model which was trained on custom objects.

### ISSUE THREAD

**Issue by AnishDeepak** [blakeblackshear/frigate#3348]
![Screenshot from 2022-06-15 10-48-16](https://user-images.githubusercontent.com/59416407/173742101-f5e062ca-be5d-4e04-93ee-a0a4686eadeb.png)
**Describe what you are trying to accomplish and why in non technical terms**
I have been trying to place my custom object detection model in frigate but getting errors. How to replace frigate object detection model with my custom object detection model. I am adding model variable in frigate.yml file which is in "frigate/frigate-has-integration/.devcontainer/ directory.
![Screenshot from 2022-06-15 10-32-01](https://user-images.githubusercontent.com/59416407/173740388-d67adc29-d6cf-496a-82cc-f288c4844434.png)
 

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by NickM-27** [comment]
Can you please provide your config as text please? There is likely a formatting error somewhere, I entered it as it appears in the screenshot and it worked

**Comment by AnishDeepak** [comment]
Hi, thank you  for the response. After I raised the issue I have tried with
ssdlite-mobiledet and mobilenetv2 models from the following github repo
from which frigate downloading the  model: google-coral/test_data: Trained
and compiled TF Lite models, and other testing data for Coral devices
(github.com) <https://github.com/google-coral/test_data/>. But when I
converted my pytorch yolov5 model's best.pt weights to tflite, I am facing
that issue. Does frigate support yolov5? or  should I only use the models
in that repository?
